src/styles/Blocks/card.scss

```diff
@@ -0,0 +1,90 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: $card-width;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  transition: transform 300ms;
+  cursor: pointer;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 3px 40px;
+  }
+
+  &__title {
+    @include fonts($size: 12px, $line: 18px);
+    color: $secondary-color;
+    margin-bottom: 4px;
+    margin-top: $removing-default-margins;
+    transition: color 300ms;
+
+    &:hover {
+      color: #34568b;
```

Comment:
This hover effect and color transition should be applied to the card, not the title. You can check this requirement on task page - point #3. At the moment you apply the hover effect to the title.

This is important because the title is much smaller than a card, so the transition won't kick in until the user hovers exactly over it, which is not ideal.

Reply:
I would argue that in this case, because he implemented the entire card, it doesn't make sense, at least to me, to highlight the text when hovering over the card. it might confuse the user and they'll think no matter where he clicks on the card, it will count as a click on the title

Reply:
If clicking on the title has a separate meaning or function then yes, it makes sense to highlight the title separately.